src/lib/cache/subCache.cpp

```diff
@@ -829,7 +829,8 @@ void subCacheItemInsert
   const std::string&                      georel,
   bool                                    blacklist,
   bool                                    onlyChanged,
-  bool                                    covered
+  bool                                    covered,
+  bool                                    notifyOnMetadataChange
```

Comment:
The new field should be added to the doc/manuals/devel/subscriptionCache.md document.